x/slashing/simulation/decoder.go

```diff
@@ -24,10 +22,7 @@ func NewDecodeStore(cdc codec.BinaryCodec) func(kvA, kvB kv.Pair) string {
 			return fmt.Sprintf("%v\n%v", infoA, infoB)
 
 		case bytes.Equal(kvA.Key[:1], types.ValidatorMissedBlockBitmapKeyPrefix):
-			var missedA, missedB gogotypes.BoolValue
-			cdc.MustUnmarshal(kvA.Value, &missedA)
-			cdc.MustUnmarshal(kvB.Value, &missedB)
-			return fmt.Sprintf("missedA: %v\nmissedB: %v", missedA.Value, missedB.Value)
+			return fmt.Sprintf("missedA: %v\nmissedB: %v\n", kvA.Value, kvB.Value)
```

Comment:
This was wrong as how the value is stored changed in https://github.com/cosmos/cosmos-sdk/pull/15580/files